supplementary_tests/src/test/etc/Globals.conf

```diff
@@ -1,111 +0,0 @@
-[Globals]
-
-	#OwnTeamColour = cyan
-	OwnTeamColour = magenta
```

Comment:
We can still have globals.conf for setting common parameters if they are independent of the agent.

Reply:
Makes sense but let's do it in when someone asks for that feature. Capabilities are not being used anyways in the engine for now.